server/odc-service/src/main/java/com/oceanbase/odc/service/datasecurity/SensitiveColumnService.java

```diff
@@ -145,6 +147,12 @@ public List<DatabaseWithAllColumns> listColumns(@NotNull Long projectId, @NotEmp
                         accessor.listBasicTableColumns(database.getName()), exists));
                 databaseColumn.setView2Columns(getFilteringExistColumns(database.getId(),
                         accessor.listBasicViewColumns(database.getName()), exists));
+                DialectType dialectType = session.getDialectType();
```

Comment:
Have you tested whether the external table can recognize sensitive columns properly?

Reply:
![image](https://github.com/user-attachments/assets/313f08fc-f796-4bd2-a950-6af22c9d2c72)
![image](https://github.com/user-attachments/assets/624ebc89-a28c-4a78-ae2b-db0bc788b988)
yes